### ISSUE TITLE
Customize left/right click on icon

### ISSUE THREAD

**Issue by nazar-pc** [nicklan/pnmixer#169]
It is possible to customize middle click already, but left/right would be also useful, especially since it is more convenient to scroll over icon to change volume already and popup menu is not very useful.

I'd like to open pavucontrol on left click for instance.

**Comment by hasufell** [comment]
Right-click is kinda standard for context menu, why would you want to customize that?

**Comment by nazar-pc** [comment]
Yeah, probably not right click, but left click for sure.
